src/components/singleProductPage/SingleProductPage.jsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import Slider from "react-slick";
+import "./singleProductPage.scss";
+
+export default function SingleProductPage(props) {
+  // These varaibles need to be replaced by the real data
+  const title = props.props.title;
+  const imgs = props.props.images;
+  const brand = props.props.brand;
+  const price = props.props.price;
+  const description = "Health, soft and fast running shoes";
+  const hasSize = true;
+  const sizes = [41, 42, 43, 44];
+
+  const PRODUCT_IMGS_SLIDER_SETTING = {
```

Comment:
usually if we have it inside a function we don't use the all caps here, its fine to keep it in here because its a function of `imgs`. just make the variable camel case